README.md

```diff
@@ -86,6 +86,10 @@ document.addEventListener('WebComponentsReady', function() {
 
 For more information about CSS variables and how they work with MyUW Web Components, [reference the styles component](https://github.com/myuw-web-components/myuw-app-styles "reference the styles component")
 
+### Releasing
+Update package.json and package-lock.json with a new version (based on the appropriate selection: `patch, minor, or major`.
```

Comment:
This is correct and will work. One way to do this version updating is with [the `npm version` command](https://docs.npmjs.com/cli/version).